apps/synapse-portal-framework/src/components/elportal/ELGettingStarted.tsx

```diff
@@ -54,53 +55,20 @@ const ELGettingStarted = () => {
         gap: { xs: '50px', md: '80px' },
       }}
     >
-      <Box
+      <PortalSectionHeader
+        reverseButtonAndText
+        title="Getting Started"
+        summaryText="We provide all the help you need for navigating the portal and
+        accelerating your research."
+        buttonText="Visit Our Help Section"
+        link="https://help.eliteportal.org/help/"
         sx={{
-          borderTop: '3px solid #ffffff88',
-          py: '20px',
+          '*': { color: '#FFF', borderColor: 'rgba(255, 255, 255, 0.40)' },
```

Comment:
Ditto on lifting the `color` style up and using `'primary.contrastText`. 

If you could make the `borderColor` style more specific to the headline element, that would be great too (not all borders in this component should be this color)